app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
+  end
+
+  def create
+    @game = Game.create(game_params)
+    @game.update_attribute(white_player_id, current_user.id)
```

Comment:
There are a number of problems here:

- Expecting `create` to fail and using it anyway is not a good idea because it will connect to the DB and create a transaction that will then need to be rolled back. If you want to build a model object without saving it, you should just use `new`.
- `update_attribute` is a dangerous method because it bypasses validations (http://stackoverflow.com/questions/2778522/rails-update-attribute-vs-update-attributes). Unless you're totally sure you want that behavior, just use `update_attributes` instead.
- There are validations on the game model that could cause record creation to fail, so you should check for this scenario and deal with a validation failure appropriately.